### PR TITLE
[SINGULAR] PEPPER-38 Add visual e2e tests for adult dependent enrollment

### PR DESCRIPTION
**Pull request by alextanlan**

### CODE REVIEW

**Review by aweng98 — DISMISSED** [review]
Sorry for the delay. Looks good to me.

File: playwright-e2e/tests/singular/enrollment/adult-dependent-enrollment-visual.spec.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    await whoHasVentricleHeartDefect.uncheck(WHO.SomeoneElse);
+    await whoHasVentricleHeartDefect.check(WHO.TheDependantBeingEnrolled);
+    await page.waitForResponse((resp) => resp.url().includes('/answers') && resp.status() === 200);
+    let screenshotSuccess = await whoHasVentricleHeartDefect.toLocator().screenshot();
+    expect(screenshotSuccess).toMatchSnapshot('whoHasVentricleHeartDefect-no-errors.png');
+  });
+
+  // Age validation: should be 18 y.o. or more
+  test('should validate adult dependent age @visual @enrollment @singular', async ({ page }) => {
+    const enrollMyAdultDependentPage = new EnrollMyAdultDependentPage(page);
```
Comment (juanseba21) [inline playwright-e2e/tests/singular/enrollment/adult-dependent-enrollment-visual.spec.ts]: It seems that we can move this line as part of the beforeEach block (it is presenting on every test).

Comment (alextanlan) [reply playwright-e2e/tests/singular/enrollment/adult-dependent-enrollment-visual.spec.ts]: Fixed.

**Review by juanseba21 — DISMISSED** [review]

File: playwright-e2e/pages/singular/enrollment/enroll-my-adult-dependent-page.ts [hunk 2]
```diff
@@ -15,8 +15,8 @@ export default class EnrollMyAdultDependentPage extends SingularPage {
   /**
    * Question: How old is your dependent?
    */
-  howOldIsYourDependent(): Input {
-    return new Input(this.page, { ddpTestID: 'answer:ENROLLING_DEPENDENT_AGE' });
+  howOldIsYourDependent(): Question {
+    return new Question(this.page, { prompt: 'How old is your dependent?' });
```
Comment (aweng98) [inline playwright-e2e/pages/singular/enrollment/enroll-my-adult-dependent-page.ts]: Oh, so we can't use `data-test-id` for this question across different enrollment workflow? That'd be not ideal. If you need a `Question` object, you can use `toQuestion()`.

**Review by aweng98 — DISMISSED** [review]
LGTM. 
One thing you still need, run this test on Docker image to generate `-linux.png` files.

**Review by juanseba21 — DISMISSED** [review]

File: playwright-e2e/pages/singular/enrollment/enroll-my-adult-dependent-page.ts (hunk 2 above)
Comment (alextanlan) [reply playwright-e2e/pages/singular/enrollment/enroll-my-adult-dependent-page.ts]: Fixed in https://github.com/broadinstitute/ddp-angular/pull/1830/commits/a81ae9804a7f4ecf5e10fdf81210ec9568e28710

**Review by aweng98 — APPROVED** [review]

**Review by juanseba21 — APPROVED** [review]

**Pull request merged**